modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -28,7 +27,7 @@ The *Display name* and *Description* fields provide supplementary information an
 +
 [source,terminal]
 ----
-$ oc label namespace bookinfo istio-injection=enabled
+$ oc label namespace bookinfo istio-discovery=enabled istio-injection=enabled 
```

Comment:
In addition to updating this command, we should probably update the text preceding to note the addition, for example: "Apply the {Istio} **discovery selector and** injection label to the `bookinfo` namespace by entering the following command at the CLI:

Reply:
Made this change. Thanks!